### PR TITLE
feature: add tasks create actions in client pkg of golang sdk

### PR DESCRIPTION
**Pull request by zgfh**
feature: add tasks create actions in client pkg of golang sdk

### Ⅰ. Describe what this PR did
add peers implementation in sdk：

POST /tasks

### Ⅱ. Does this pull request fix one issue?

this fixes one part of #366

### Ⅳ. Describe how to verify it
none

### Ⅴ. Special notes for reviews
none

### CODE REVIEW

File: client/task_create.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+package client
+
+import (
+	"context"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+// TaskCreate creates a peer node in supernode.
```
Comment (yeya24) [inline client/task_create.go]: Update this commend.

Comment (zgfh) [reply client/task_create.go]: updated

Comment (yeya24) [reply client/task_create.go]: remote `node`

**Comment by allencloud** [comment]
Actually, in the current supernode, we still use the API endpoint of java version. We are still coupling the objects behind the API(not in the restful way), like in the https://github.com/dragonflyoss/Dragonfly/blob/master/supernode/server/router.go#L28-L32

What is our plan that starts to implement the restful API, including API server routes and API client?  @yeya24 @Starnop @lowzj

File: client/task_create.go (hunk 1 above)
Comment (zgfh) [reply client/task_create.go]: @yeya24 very sorry, updated

**Comment by zgfh** [comment]
if we have a plan to implement the restful API of tasks, I can help if need

**Comment by allencloud** [comment]
LGTM

**Pull request merged**